lib/local-links-manager/check_links/link_status_updater.rb

```diff
@@ -3,29 +3,27 @@
 module LocalLinksManager
   module CheckLinks
     class LinkStatusUpdater
```

Comment:
The one con of that is that the link checker is months away from discovery let alone actually building it. It will come after Lolotron V2 and mid (maybe after depending . . . ) migration